packages/select/src/ListMenu.tsx

```diff
@@ -63,6 +63,7 @@ const ListMenu = React.forwardRef<HTMLUListElement, ListMenuProps>(
       scrollContainer,
       portalClassName,
       popoverZIndex,
+      ...rest
```

Comment:
I don't know if we should add this yet. Consumers shouldn't be able to change stuff like `adjustOnMutation`, `spacing`, `onClick`, `align`. Not sure why it wasn't omitted. If we do keep this maybe we update the type to omit these props?

Reply:
Hm okay. I'll just remove it for now then - since we're not separating Popover's props from the Button's, supporting all props now would be kind of a hassle. That can come as a part of a larger effort later.